packages/core/src/scenes/GeneratorScene.ts

```diff
@@ -48,6 +48,7 @@ export abstract class GeneratorScene<T>
   public readonly variables: Variables;
   public random: Random;
   public creationStack?: string;
+  public readonly belowPrevious: SimpleSignal<boolean, this>;
```

Comment:
Scenes should not use signals directly as it may lead to memory leaks.
You should store the raw `SignalValue` instead, and use `unwrap` to access it whenever necessary.

Reply:
It would also be a goof idea to reset this property in the `reset` method

Reply:
Was not aware of that, good catch. Updated to store the raw `SignalValue`.